sql/1/openfire.sql

```diff
@@ -620,8 +620,8 @@ COPY public.ofid (idtype, id) FROM stdin;
 --
 
 COPY public.ofmucaffiliation (roomid, jid, affiliation) FROM stdin;
-1	admin@localhost.example	10
-2	admin@localhost.example	10
+1	admin@xmpp1.localhost.example	10
+2	admin@xmpp1.localhost.example	10

```

Comment:
Does one of these need to be `xmpp2` rather than `xmpp1`? What _exactly_ is the setup here? Do we have two XMPP domains, each with one room in it, or do we have two domains, where one has one or more rooms?

Reply:
We have the later, two XMPP domains, one of which (xmpp1.localhost.example) has two MUC rooms.

Would you rather have one MUC room in each?

Reply:
Nope, this is fine for me. It seemed like a potential typo / copy/paste mistake, which is why I flagged it.